internal/biz/resources/resources_test.go

```diff
@@ -250,6 +250,8 @@ func TestCreateNewResource(t *testing.T) {
 }
 
 func TestCreateNewResource_ConsistencyToken(t *testing.T) {
+	// TODO: Follow up with leads on which consistency to support in v1beta1
+	t.Skip("TODO: Test is failing because sync consistency flow was ripped out")
```

Comment:
👍 I plan on adding tests to cover these sync/async cases with the added `wait_for_sync` option.